packages/bull-vault/src/FlashBull.sol

```diff
@@ -18,17 +18,16 @@ contract FlashBull {
 
     /// @dev enum to differentiate between Uniswap swap callback function source
     enum FLASH_SOURCE {
+        GENERAL_SWAP,
```

Comment:
What is this going to be used for?

Reply:
Selling borrowed USDC for ETH, added in this PR but didn't use it